packages/main/src/Select.js

```diff
@@ -504,11 +540,87 @@ class Select extends UI5Element {
 		&& this.responsivePopover.opened ? "-1" : "0";
 	}
 
+	get classes() {
+		return {
+			popoverValueState: {
+				"ui5-valuestatemessage-root": true,
+				"ui5-valuestatemessage--success": this.valueState === ValueState.Success,
+				"ui5-valuestatemessage--error": this.valueState === ValueState.Error,
+				"ui5-valuestatemessage--warning": this.valueState === ValueState.Warning,
+				"ui5-valuestatemessage--information": this.valueState === ValueState.Information,
+			},
+		};
+	}
+
+	get styles() {
+		return {
+			popoverHeader: {
+				"width": `${this.offsetWidth}px`,
+			},
+			responsivePopoverHeader: {
+				"display": this.options.length && this._listWidth === 0 ? "none" : "inline-block",
+				"width": `${this.options.length ? this._listWidth : this.offsetWidth}px`,
+				"padding": "0.5625rem 1rem",
+			},
+			dialogHeader: {
```

Comment:
Can't this be "class", usually in "styles" we only set dynamic properties

Reply:
Done